paddlenlp/transformers/configuration_utils.py

```diff
@@ -678,13 +628,7 @@ def save_pretrained(self, save_directory: Union[str, os.PathLike], **kwargs):
         logger.info(f"Configuration saved in {output_config_file}")
 
     @classmethod
-    def from_pretrained(
-        cls,
-        pretrained_model_name_or_path: Union[str, os.PathLike],
-        from_hf_hub: bool = False,
-        cache_dir: Optional[str] = None,
-        **kwargs
-    ) -> PretrainedConfig:
+    def from_pretrained(cls, pretrained_model_name_or_path: Union[str, os.PathLike], **kwargs) -> PretrainedConfig:
```

Comment:
All parameters other than `pretrained_model_name_or_path` are passed in through `kwargs`